php/libraries/User.class.inc

```diff
@@ -317,6 +317,21 @@ class User extends UserPermissions
     }
 
     /**
+     * Checks if the user is in at least one study site
+     *
+     * @return boolean
+     */
+    function isInOneStudySite()
```

Comment:
I'm not a fan of the prefix "isInOne" because it sounds like it means must have exactly one. What about "hasStudySite"?